scripts/remove-default-apps.ps1

```diff
@@ -34,13 +34,14 @@ $apps = @(
     "Microsoft.ZuneMusic"
     "Microsoft.ZuneVideo"
     "microsoft.windowscommunicationsapps"
+    "Microsoft.MinecraftUWP"
 
-    # apps from Windows 8 upgrade
+    # non-Microsoft

```

Comment:
Changed this because the following apps appear also on a clean install of Win10 Home